modules/services/analytics/src/main/java/au/com/shiftyjelly/pocketcasts/analytics/AnalyticsEvent.kt

```diff
@@ -267,4 +267,5 @@ enum class AnalyticsEvent(val key: String) {
     PLAYBACK_EFFECT_TRIM_SILENCE_TOGGLED("playback_effect_trim_silence_toggled"),
     PLAYBACK_EFFECT_TRIM_SILENCE_AMOUNT_CHANGED("playback_effect_trim_silence_amount_changed"),
     PLAYBACK_EFFECT_VOLUME_BOOST_TOGGLED("playback_effect_volume_boost_toggled"),
+    FILTER_LIST_REORDERED("filter_list_reordered"),
```

Comment:
I realized after merging that `FILTER_LIST_REORDERED` event can be grouped with the `Filters screen` events. Can you take care of it in your upcoming filters PR?

Reply:
> Can you take care of it in your upcoming filters PR?

Yes, of course! Thanks for catching that. 🙇